src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menList = people.filter(person => (
+    century
+      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+      : person.sex === 'm'
+  ));
+
+  const sumOfAge = menList.reduce((sum, currentMan) => {
+    const currentManAge = currentMan.died - currentMan.born;
+
+    return sum + currentManAge;
+  }, 0);
+
+  const averageAge = sumOfAge / menList.length;
```

Comment:
let's create separate function that will accept people array and will return the average age value